docs/topics/compatibility-guide-19.md

```diff
@@ -41,6 +41,21 @@ perspective
 > - 1.8.0: raise the warning to an error
 -->
 
+### Remove language version 1.3
+
+> **Issue**: [KT-61111](https://youtrack.jetbrains.com/issue/KT-61111/Remove-language-version-1.3)
+>
+> **Component**: Core language
+>
+> **Incompatible change type**: source
+>
+> **Short summary**: Kotlin 1.9 introduces language version 1.9 and removes support for language version 1.3. 
```

Comment:
```suggestion
> **Short summary**: Kotlin 1.9.0 introduces language version 1.9 and removes support for language version 1.3. 
```

Reply:
Thanks for the suggestion! We've used "Kotlin 1.9" in the rest of the guide and in previous versions. This covers 1.9.0 and 1.9.20, etc. It's not my favorite notation so I hope we can change it once we move to 2.x So I won't apply this for now for consistency.